### PR TITLE
fix: audit Basic CSS block

### PR DESCRIPTION
**Pull request by zairahira**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

The following fixes have been made:

#### Video: How do width and height work? (make sure to also talk about min/max width and height here)
- fix: updated question 3 as `rem` hasn't been taught.
#### Video: What are the different types of CSS combinators?
- fix: last question, removed code block.
#### Video: How does inline-block work and how does it differ from inline and block elements?
- fix: last question, fix typo

### Basic CSS review
- Mentioned "structure of using a CSS rule" as it was asked in final quiz.
- Shared an example of inline css
- Added padding and margin short hand property																		

### Basic CSS Quiz
- Convert blockline -> block-level

#### Video: How do you space list items using margin or line-height?
- Fix: Removed duplicate questions

_new changes_

### backgrounds, borders CSS Review page

- Added allowed and default values taught in lectcures for properties asked in quiz.

### backgrounds, borders CSS Quiz

- updated question as `border-width` not taught
- updated question as `border-collapse` property not taught
- updated options as `lower-greek` was not discussed

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/lecture-what-is-css/672acc03c257524d6a5151e8.md [hunk 1]
```diff
@@ -98,15 +98,15 @@ CSS allows various units to set dimensions.
 
 ---
 
-`px`, `vh`, and `vw`.
+`pxx`, `depth`, and `radius`.
 
 ### --feedback--
 
 CSS allows various units to set dimensions.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lecture-what-is-css/672acc03c257524d6a5151e8.md:105]: this feedback doesn't explain why the answer is wrong nor hints to the right one, maybe it needs some expanding?

File: curriculum/challenges/english/25-front-end-development/quiz-css-backgrounds-and-borders/66ed8fd7f45ce3ece4053eb0.md [hunk 2]
```diff
@@ -193,23 +193,24 @@ The background stays fixed while the content scrolls
...
 ---
 
-`groove`
+`zig zag`
 
 #### --answer--
 
-`triple`
+`dotted`
+
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/quiz-css-backgrounds-and-borders/66ed8fd7f45ce3ece4053eb0.md:214]:
```suggestion
`dotted`

```
there is an accidental extra line

File: curriculum/challenges/english/25-front-end-development/review-css-backgrounds-and-borders/671a88d636cebc5fbd407b78.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 id: 671a88d636cebc5fbd407b78
-title: CSS Backgrounds and Borders Review
+title: Lists, Links, CSS background and Borders Review
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/review-css-backgrounds-and-borders/671a88d636cebc5fbd407b78.md:3]:
```suggestion
title: Lists, Links, CSS Background and Borders Review
```

everything is capitalised in titles usually

File: client/i18n/locales/english/intro.json [hunk 4]
```diff
@@ -2063,7 +2063,7 @@
         ]
       },
       "review-css-backgrounds-and-borders": {
-        "title": "CSS Backgrounds and Borders Review",
+        "title": "Lists, Links, CSS background and Borders Review",
```
Comment (ilenia-magoni) [inline client/i18n/locales/english/intro.json:2066]:
```suggestion
        "title": "Lists, Links, CSS Background and Borders Review",
```

File: client/src/pages/learn/full-stack-developer/review-css-backgrounds-and-borders/index.md [hunk 5]
```diff
@@ -1,9 +1,9 @@
 ---
-title: Introduction to the CSS Backgrounds and Borders Review
+title: Introduction to the Lists, links, CSS background and Borders Review
```
Comment (ilenia-magoni) [inline client/src/pages/learn/full-stack-developer/review-css-backgrounds-and-borders/index.md:2]:
```suggestion
title: Introduction to the Lists, links, CSS Background and Borders Review
```

File: client/src/pages/learn/full-stack-developer/review-css-backgrounds-and-borders/index.md [hunk 6]
```diff
@@ -1,9 +1,9 @@
...
 block: review-css-backgrounds-and-borders
 superBlock: full-stack-developer
 ---
 
-## Introduction to the CSS Backgrounds and Borders Review
+## Introduction to the Lists, links, CSS background and Borders Review
```
Comment (ilenia-magoni) [inline client/src/pages/learn/full-stack-developer/review-css-backgrounds-and-borders/index.md:7]:
```suggestion
## Introduction to the Lists, Links, CSS Background and Borders Review
```

File: curriculum/challenges/_meta/review-css-backgrounds-and-borders/meta.json [hunk 7]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "CSS Backgrounds and Borders Review",
+  "name": "Lists, links, CSS background and Borders Review",
```
Comment (ilenia-magoni) [inline curriculum/challenges/_meta/review-css-backgrounds-and-borders/meta.json:2]:
```suggestion
  "name": "Lists, Links, CSS Background and Borders Review",
```

File: curriculum/challenges/_meta/review-css-backgrounds-and-borders/meta.json [hunk 8]
```diff
@@ -8,7 +8,7 @@
   "challengeOrder": [
     {
       "id": "671a88d636cebc5fbd407b78",
-      "title": "CSS Backgrounds and Borders Review"
+      "title": "Lists, Links, CSS background and Borders Review"
```
Comment (ilenia-magoni) [inline curriculum/challenges/_meta/review-css-backgrounds-and-borders/meta.json:11]:
```suggestion
      "title": "Lists, Links, CSS Background and Borders Review"
```

File: curriculum/challenges/english/25-front-end-development/lecture-what-is-css/672acc03c257524d6a5151e8.md (hunk 1 above)
Comment (zairahira) [reply curriculum/challenges/english/25-front-end-development/lecture-what-is-css/672acc03c257524d6a5151e8.md]: done!

**Review by majestic-owl448 — APPROVED** [review]

**Review by scissorsneedfoodtoo — APPROVED** [review]
Thanks for catching all of these issues, @zairahira, and for the hard work you and @ilenia-magoni put into reviewing and fixing them all.

The one thing I noticed is that we still have one quiz with the old title "CSS Backgrounds and Borders Quiz". It's not a show stopper at all, though.

If we have time, we can follow up with another PR to update that quiz title, and possibly go the extra mile to add a serial comma after the word "and" so it's "Lists, Links, CSS Background, and Borders...", but again, it's a minor detail.

**Pull request merged**